server/service/integration_live_queries_test.go

```diff
@@ -613,87 +686,134 @@ func (s *liveQueriesTestSuite) TestLiveQueriesRestFailsToCreateCampaign() {
 	assert.Equal(t, 0, liveQueryResp.Summary.RespondedHostCount)
 	require.NotNil(t, liveQueryResp.Results[0].Error)
 	assert.Contains(t, *liveQueryResp.Results[0].Error, "Query 999 was not found in the datastore")
+
+	oneLiveQueryRequest := runOneLiveQueryRequest{
+		HostIDs: []uint{888},
+	}
+	oneLiveQueryResp := runOneLiveQueryResponse{}
+	s.DoJSON("POST", fmt.Sprintf("/api/latest/fleet/queries/%d/run", 999), oneLiveQueryRequest, http.StatusNotFound, &oneLiveQueryResp)
+	assert.Equal(t, 0, oneLiveQueryResp.RespondedHostCount)
+
```

Comment:
We could also test with an existing query but unexisting host.

Reply:
Yes, that's a good idea.